lib/tests/packages_detect.rb

```diff
@@ -2,20 +2,23 @@
 
 module Homebrew
   module Tests
-    class FormulaeDetect < Test
-      attr_reader :testing_formulae, :added_formulae, :deleted_formulae
+    class PackagesDetect < Test
+      attr_reader :testing_formulae, :added_formulae, :deleted_formulae, :testing_packages, :added_packages,
+                  :deleted_packages
 
       def initialize(argument, tap:, git:, dry_run:, fail_fast:, verbose:)
         super(tap: tap, git: git, dry_run: dry_run, fail_fast: fail_fast, verbose: verbose)
 
         @argument = argument
         @added_formulae = []
+        @added_packages = []
```

Comment:
I'm confused as to why `formulae` and `packages` are both needed here. Feels like this should be `formulae` and `casks` or just `packages`.

Reply:
I wrote it as independent as possible. It's not entirely clear to me how deprecations would work here, so I figured I'd keep everything separate.

Reply:
@SMillerDev Yeh, I think this is a bit too messy I'm afraid to merge as-is.

Reply:
I understand that, what way would you prefer we go though? Deprecate the old way and just have packages? Or keep formulae and casks?

Reply:
Either:
- migrate all uses of these variables and the `attr_reader` to `*_packages`
or:
- have `*_casks` variables and readers instead of `packages`.